packages/ozone/src/lexicon/lexicons.ts

```diff
@@ -10831,6 +10831,24 @@ export const schemaDict = {
             description:
               'Timestamp referencing the first moderation status impacting event was emitted on the subject',
           },
+          recordUpdatedAt: {
+            type: 'string',
+            format: 'datetime',
+            description:
+              'Last update timestamp of the record the subject is associated with',
```

Comment:
I feel like the language here is quite convoluted. would love suggestions to make it clearer.

Reply:
Perhaps something along the lines of: "Timestamp at which the record or account was last updated."